### DISCUSSION TITLE
根据文档按需引入组件库，showToast等函数api，eslint会报'showToast' is not defined

### DISCUSSION THREAD

**Discussion by paker777** [youzan/vant#12775] [Q&A]
import vue from '@vitejs/plugin-vue'
import vueDevTools from 'vite-plugin-vue-devtools'

import eslintPlugin from 'vite-plugin-eslint'
import AutoImport from 'unplugin-auto-import/vite'
import Components from 'unplugin-vue-components/vite'
import { VantResolver } from '@vant/auto-import-resolver'
import vitePluginStyleVwLoader from 'vite-plugin-style-vw-loader'
export default function createVitePlugins() {
  return [
    vitePluginStyleVwLoader({ viewportWidth: 375 }), // 该插件需要放在vue()之前
    vue(),
    vueDevTools(),
    eslintPlugin({
      include: ['src/**/*.js', 'src/**/*.vue', 'src/*.js', 'src/*.vue']
    }),
    AutoImport({
      imports: ['vue', 'vue-router', 'pinia'],
      dts: './types/auto-import.d.ts',
      eslintrc: {
        enabled: true,
        filepath: './.eslintrc-auto-import.json'
      },
      resolvers: [VantResolver()]
    }),
    Components({
      dirs: ['src/components'], // 自动导入全局组件
      resolvers: [VantResolver()] // 自动按需导入vant组件
    })
  ]
}

**Comment by LemonLSJ** [comment]
在tsconfig.app.json文件的include中引入auto-imports和components两个声明文件就不报错了
<img width="282" alt="iShot_2024-10-13_04 22 12" src="https://github.com/user-attachments/assets/5e325607-fb28-4e08-84c6-a975d3abd9e3">
